modules/library/metadata/src/main/java/org/geotools/util/PartiallyOrderedSet.java

```diff
@@ -14,10 +14,9 @@
  *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  *    Lesser General Public License for more details.
  */
-package org.geotools;
+package org.geotools.util;
```

Comment:
So what is the reason to change the package here? That may (or may not) break down stream code...

Reply:
I actually moved the class from `library/xml` to `library/metadata`. The `util´ just seemed the most fitting in Metadata. Note that there is currently no class in `org.geotools` in Metadata.